cadence/transactions/bridge/nft/bridge_nft_to_any_evm_address.cdc

```diff
@@ -0,0 +1,98 @@
+import "FungibleToken"
+import "NonFungibleToken"
+import "ViewResolver"
+import "MetadataViews"
+import "FlowToken"
+
+import "ScopedFTProviders"
+
+import "EVM"
+
+import "FlowEVMBridge"
+import "FlowEVMBridgeConfig"
+import "FlowEVMBridgeUtils"
+
+/// Bridges an NFT from the signer's collection in Cadence to the named recipient in EVM.
+///
+/// NOTE: This transaction also onboards the NFT to the bridge if necessary which may incur additional fees
+///     than bridging an asset that has already been onboarded.
+///
+/// @param nftContractAddress: The Flow account address hosting the NFT-defining Cadence contract
+/// @param nftContractName: The name of the NFT-defining Cadence contract
+/// @param id: The Cadence NFT.id of the NFT to bridge to EVM
+/// @param recipient: The hex-encoded EVM address to receive the NFT
+///
+transaction(nftContractAddress: Address, nftContractName: String, id: UInt64, recipient: String) {
+    
+    let nft: @{NonFungibleToken.NFT}
+    let requiresOnboarding: Bool
+    let scopedProvider: @ScopedFTProviders.ScopedFTProvider
+    
+    prepare(signer: auth(CopyValue, BorrowValue, IssueStorageCapabilityController, PublishCapability, SaveValue) &Account) {        
+        /* --- Retrieve the NFT --- */
+        //
+        // Borrow a reference to the NFT collection, configuring if necessary
+        let viewResolver = getAccount(nftContractAddress).contracts.borrow<&{ViewResolver}>(name: nftContractName)
+            ?? panic("Could not borrow ViewResolver from NFT contract")
+        let collectionData = viewResolver.resolveContractView(
+                resourceType: nil,
+                viewType: Type<MetadataViews.NFTCollectionData>()
+            ) as! MetadataViews.NFTCollectionData? ?? panic("Could not resolve NFTCollectionData view")
+        let collection = signer.storage.borrow<auth(NonFungibleToken.Withdraw) &{NonFungibleToken.Collection}>(
+                from: collectionData.storagePath
+            ) ?? panic("Could not access signer's NFT Collection")
+
+        // Withdraw the requested NFT & calculate the approximate bridge fee based on NFT storage usage
+        let currentStorageUsage = signer.storage.used
+        self.nft <- collection.withdraw(withdrawID: id)
+        let withdrawnStorageUsage = signer.storage.used
+        var approxFee = FlowEVMBridgeUtils.calculateBridgeFee(
+                bytes: currentStorageUsage - withdrawnStorageUsage
+            ) * 1.10
```

Comment:
Why 1.10?

Reply:
The extra 10% is a buffer, but tbh the exact buffer of 10% is about the limit of what was successful in my testing - nothing very rigorous or formal.

When I was testing, I found that the storage used as calculated on withdrawal was different from storage used when escrowing. This is likely because the bridge may additionally store extra pieces of information that weren't present in the source NFT collection when it was withdrawn.